Mathlib/Algebra/MvPolynomial/PDeriv.lean

```diff
@@ -72,6 +72,13 @@ theorem pderiv_monomial {i : σ} :
   · rw [Finsupp.not_mem_support_iff] at hi; simp [hi]
   · simp
 
+lemma X_mul_pderiv_monomial {i : σ} {m : σ →₀ ℕ} {r : R} :
```

Comment:
Maybe some of the variable should be explicit?

Reply:
Since each of `i` `m` and `r` appear in both side of the equality, I think this is good as is. You can see I don't need to specify any implicit arguments when rewriting with it.